development/gcp/cloud-functions/serviceaccountcleaner/README.md

```diff
@@ -0,0 +1,35 @@
+# Cleanup service account secrets using Cloud Function
+
+## Overview
+
+Cloud Function creates a new key for a GCP service account and updates the required secret data. The function is triggered by a  Pub/Sub message sent by a secret stored in Secret Manager.
+
+1. Cloud Scheduler starts the Cloud Function.
+2. For each secret stored in Secret Manager:
```

Comment:
The steps below doesn't contain steps for creating a new key for GCP service account. Or the steps or Overview section is wrong.

Reply:
Overview section has been corrected